src/gui/rss/rsswidget.cpp

```diff
@@ -54,6 +54,50 @@
 #include "feedlistwidget.h"
 #include "ui_rsswidget.h"
 
+namespace
+{
+    void convertRelativeUrlToAbsolute(QString &html, const QString &baseUrl);
+    
```

Comment:
```suggestion
```

Reply:
Although compiler should auto generate the declaration from definition, why should we remove the explicit declaration for the function? Is it considered as a bad practice for coding?

Reply:
Explicit declaration is useful only when function is used in the code before its definition appears.

Reply:
Explicit declaration immediatelly followed by definition is meaningless anyway.

Reply:
OK. Considering the function is only called in this scope, explicit declaration is redundant.